webpack.config.shared.js

```diff
@@ -57,7 +57,11 @@ module.exports = {
     exclude: /node_modules/,
     loader: 'babel-loader',
     query: {
-      "presets": ["latest", "react"],
+      "presets": [
+        ["env", { "modules": false }],
```

Comment:
because tree shaking needs to be able to perform static analysis of ES6 modules to work